certification/pyxis/pyxis.go

```diff
@@ -127,6 +127,53 @@ func (p *pyxisClient) getImage(ctx context.Context, dockerImageDigest string) (*
 	return &data.Data[0], nil
 }
 
+// updateImage updates the certification status to true for a given architecture of an image
+// this method's purpose is to account for pyxis errors in a given check (HTTP 500 as an example), where the user
+// needs to re-run preflight to get a valid response from pyxis
+func (p *pyxisClient) updateImage(ctx context.Context, imageID string, architecture string) (*CertImage, error) {
+	// instantiating a new certImage struct, if updateImage is called we know that we need to set the certified status to true
+	certImage := &CertImage{
+		ID:           imageID,
+		Architecture: architecture,
+		Certified:    true,
+	}
```

Comment:
I don't love auto-setting `Certified: true`. If we're going to do this, we should rename this method.

I know it's a private method, but I would still rather the method name align with this predefined behavior.

Reply:
Seems like you and @bcrochet agree this method should be refactored to just take a `&CertImage` and submit do the proper work. I'll make that change.